04/app.js

```diff
@@ -0,0 +1,37 @@
+const arr = [];
+
+const createArr = function (min, max) {
+  for (let i = 0; i < 30; i++) {
```

Comment:
Lepiej byłoby stworzyć zmienną wew. funkcji - wtedy byłabo to zmienna lokalna, a nie globalna.